packages/objects-horizontal-list/_horizontal-list.scss

```diff
@@ -16,6 +16,7 @@ $scales-objects-class-namespace: null !default;
 // Available Variables
 ///
 $horizontal-list-delimiter: ',\00A0' !default;
+$horizontal-list-spacing: 1rem !default;
```

Comment:
I think we should do the default 0 and then have modifiers for `tiny`, `small`, `medium`, `large`, `huge` like other patterns have.

Reply:
I was following base-lists, which just has the one, should we update that one too?

Reply:
Base lists doesn't have anything for spacing in it. Just the margin that pushes the whole list over a ways. maybe this pattern just needs one variation. So leave the base class completely alone and add a modifier for adding space that uses the variable.